apps/cms/lib/partial/paragraph/code_embed.ex

```diff
@@ -0,0 +1,30 @@
+defmodule CMS.Partial.Paragraph.CodeEmbed do
+  @moduledoc """
+  Represents a CodeEmbed Paragraph in the CMS.
+  Body is raw HTML from CMS, and is not scrubbed or rewritten.
+  """
+
+  alias CMS.Partial.Paragraph.ColumnMultiHeader
+  alias Phoenix.HTML
+
+  import CMS.Helpers, only: [field_value: 2, parse_paragraphs: 3]
+
+  defstruct header: nil,
+            right_rail: false,
+            body: HTML.raw("")
+
+  @type t :: %__MODULE__{
+          header: ColumnMultiHeader.t() | nil,
+          right_rail: boolean,
```

Comment:
Pretty much all of our paragraphs can be set to show in the right sidebar of the page on the front-end. This allows authors to put a small code snippet/embed in the right rail of the template, similar to other paragraphs.